config/locales/en.yml

```diff
@@ -446,10 +447,12 @@ en:
       rotate_other_certificates: You can rotate other certificates straight away - you do not need to wait for the email.
       received_email: Once you've received the email, delete the old %{usage} key and certificate from your %{component} configuration.
       received_email_to_replace: Once you've received the email, replace the old %{usage} key with the new one in your %{component} configuration.
+      received_email_to_promote: Once you've received the email, promote your secondary %{usage} key and certificate to the primary position and delete the old one in your %{component} configuration.
       rotate_more: Rotate more certificates
       msa_add_new_key_and_certificate: Add the new signing key and certificate to your Matching Service Adapter (MSA) configuration.
       tell_us_to_stop_using_old_certificate: Tell GOV.UK Verify to stop using your old certificate using this tool.
       configuration_warning: You must update your %{component} configuration by %{date}, or your service’s connection to GOV.UK Verify will break.
       certificate_rotations_are_independent: You can rotate other certificates while you wait for the email - certificate rotations are independent of each other.
       tell_verify: Tell GOV.UK Verify to stop using your old (or 'secondary') certificate using this tool.
+      tell_verify_old: Tell GOV.UK Verify to stop using your old certificate using this tool.
```

Comment:
I wonder if `using this tool` could change to something along the lines of `stop using your old certificate by managing it through the self service dashboard` and then linking to the dashboard?

Reply:
I'll not merge until Beth or someone reviewed from the content point of view.. I tried my best 🤷‍♂ 😆